### PR TITLE
Document merge and write actions

### PR DESCRIPTION
**Pull request by sblausten**
#### :heavy_check_mark: Checklist

- [ ] Added tests for new functionality and regression tests for bug fixes
- [ ] Added changeset (run `yarn changeset` in the root)
- [ ] Screenshots of before and after attached (for UI changes)
- [ ] Added or updated documentation (if applicable)

### CODE REVIEW

File: plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md [hunk 1]
```diff
@@ -478,3 +493,61 @@ spec:
...
 
 // output: `"\"hello\": \"world\""`
+
+### Merge Json
+
+Action: `roadiehq:utils:json:merge`
+
+**Required params:**
+
+- path: The file path for the json you want to edit.
```
Comment (Xantier) [inline plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md:503]: JSON should be all caps in most places

**Review by Xantier — DISMISSED** [review]

File: plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md [hunk 2]
```diff
@@ -478,3 +493,61 @@ spec:
...
+- content: The Json you want to merge in. Can be either an object or a string.
+
+```yaml
+---
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: merge-json-into-file
```
Comment (punkle) [inline plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md:511]: I dont see how this is merging? It seems to be writing a string content to a file?

Comment (sblausten) [reply plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md:511]: Ah sorry getting changes mixed up.

File: plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md [hunk 3]
```diff
@@ -478,3 +494,170 @@ spec:
...
+      name: Fetch repo
+      action: fetch:plain
+      input:
+        url: 'https://github.com/${{ parameters.org }}/${{ parameters.repository }}'
+    - id: merge
+      name: Merge JSON
+      action: roadiehq:utils:json:merge
+      input:
+        path: ${{ parameters.path }}
+        content: ${{ parameters.content }}
```
Comment (punkle) [inline plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md:559]: Can we make a second example where content is an object e.g.:

```yaml
    - id: merge
      name: Merge JSON
      action: roadiehq:utils:json:merge
      input:
        path: ${{ parameters.path }}
        content:
          hello: world
```

File: plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md [hunk 4]
```diff
@@ -478,3 +494,170 @@ spec:
...
+        message: 'RemoteURL: ${{ steps["publish-pr"].output.remoteUrl }}'
+```
+
+### Write to File
+
+**Action name:** `roadiehq:utils:fs:write`
+
+**Required params:**
+
+- path: The file path for the JSON you want to edit.
```
Comment (punkle) [inline plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md:581]: This should say:

```
- path: The file path for the content you want to write.
```

File: plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md [hunk 5]
```diff
@@ -171,9 +177,11 @@ spec:
         path: ${{ parameters.amount }}
 ```
 
-### Deserialise - `roadiehq:utils:fs:parse`
+### Deserialise
 
-This action deserialises json or yaml files in the temporary scaffolder workspace to a javascript object in memory that can then be passed to another step.
+**Action name**: `roadiehq:utils:fs:parse`
+
+This action deserialises JSON or YAML files in the temporary scaffolder workspace to a javascript object in memory that can then be passed to another step.
```
Comment (Xantier) [inline plugins/scaffolder-actions/scaffolder-backend-module-utils/README.md:184]:
```suggestion
This action deserialises JSON or YAML files in the temporary scaffolder workspace to a JavaScript object in memory that can then be passed to another step.
```

**Review by Xantier — DISMISSED** [review]

**Review by Xantier — APPROVED** [review]

**Pull request merged**